SECURITY.md

```diff
@@ -76,4 +105,31 @@ Extensions configuration setup is tied to OpenSearch settings, extensions config
 
 ## Reliability
 
-OpenSearch plugins can create cluster or node instability if incorrectly configured or by software defects.
\ No newline at end of file
+OpenSearch plugins can create cluster or node instability if incorrectly configured or by software defects.
+
+# Projects
+To stretch out the design process while fulfilling scenarios some security efforts will be tracked as longer running projects.  There will be tracking issues in github tied to the work, documentation here is justification for a project and how it ties into the security space.  Tasks will be denotated as incomplete with :negative_squared_cross_mark: `:negative_squared_cross_mark:` or completed with :white_check_mark:
+ `:white_check_mark:`.
+
+## Anomaly Detection Plugin
+Overall project is tracked with [[FEATURE] Migrate Anomaly Detector plugin to work as an Extension](https://github.com/opensearch-project/opensearch-sdk/issues/24).  By migrating this plugin it will exercise the general extensions and security specific scenarios.
+
+Additional background avaliable from [Security#1895](https://github.com/opensearch-project/security/issues/1895)
+
+### User identity [OpenSearch#3846](https://github.com/opensearch-project/OpenSearch/issues/3846) :negative_squared_cross_mark:
+Replace [commons.authuser.User](https://github.com/opensearch-project/common-utils/blob/main/src/main/java/org/opensearch/commons/authuser/User.java) the common identity object used by plugins.  The new object should be obtainable by the extension through other means than [InjectSecurity](https://github.com/opensearch-project/common-utils/blob/main/src/main/java/org/opensearch/commons/InjectSecurity.java#L69) depends on the thread context.
+
+#### Aquiring User objects [sdk#37](https://github.com/opensearch-project/opensearch-sdk/issues/37)
+When OpenSearch sents a request to an extension the identiy should be avaliable, more discussion see [Handling identity in extensions](https://github.com/opensearch-project/opensearch-sdk/issues/14).
+
+#### Resource user/role checks [sdk#40](https://github.com/opensearch-project/opensearch-sdk/issues/40)
+anomaly Detection has detectors that analyzer data and store its results so it can be inspected or alerted on, [more details](https://opensearch.org/docs/latest/monitoring-plugins/ad/index/). OpenSearch should be responsible for inspecting the user, roles, resources to ensure standard practices are used.  A permissions check API should be designed and implemented to offload this work from extensions creators.
```

Comment:
Given we're touching on RBAC here with "user, roles, resources" I think the term "permissions check API" may be better termed "access check API" or at least "access permissions check API"... open to ideas.

Reply:
> "access check API"

I like it